uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_sidebar.scss

```diff
@@ -1,3 +1,6 @@
+// Variables
+$light-gray : #D2D2D2;
```

Comment:
The variables should be moved to `sites/all/modules/civicrm/tools/extensions/civihr_tasks/uk.co.compucorp.civicrm.tasksassignments/scss/civihr/modules/_variables.scss` file.